src/familybot/lib/database.py

```
@@ -948,21 +977,99 @@ def cleanup_expired_cache():
+def _mark_migration_run(migration_name: str) -> None:
+    """Record that a named migration has been applied."""
+    conn = None
+    try:
+        conn = get_db_connection()
+        cursor = conn.cursor()
+        cursor.execute(
+            "INSERT OR IGNORE INTO migrations (name, applied_at) VALUES (?, STRFTIME('%Y-%m-%dT%H:%M:%fZ', 'NOW'))",
+            (migration_name,),
+        )
+        conn.commit()
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)